pkg/proc/linutil/regs_arm64_arch.go

```diff
@@ -0,0 +1,139 @@
+package linutil
+
+import (
+	"fmt"
+	"golang.org/x/arch/arm64/arm64asm"
+
+	"github.com/go-delve/delve/pkg/proc"
+)
+
+// Regs is a wrapper for sys.PtraceRegs.
+type ARM64Registers struct {
+	Regs     *ARM64PtraceRegs //通用寄存器
```

Comment:
You simplely copyed my old patch? If you want to improve arm64-delve, you should push after #1733 merged.

Reply:
> You simplely copyed my old patch? If you want to improve arm64-delve, you should push after #1733 merged.
We both want to take delve support for ARM64 forward and can wait for your merge .. 
i think your patch is on top of our old patch https://github.com/go-delve/delve/pull/1690

Reply:
No, the ptraceGetGRegs/ptraceSetGRegs are totally diferent, which is the fundamental of debuging, and Location func on my patch, and so on. i clear-up testsuits to passed CI.
i waited a month and  though you may gave up. 
Honestly, #1690 have so many errors that it even can't read from memory. Whatever it is an good thing that delve can support arm64 now. Now it's only a fundamental native debug, let's improve it step by step.

Reply:
> No, the ptraceGetGRegs/ptraceSetGRegs are totally diferent, which is the fundamental of debuging, and Location func on my patch, and so on. i clear-up testsuits to passed CI.
> i waited a month and though you may gave up.
> Honestly, #1690 have so many errors that it even can't read from memory. Whatever it is an good thing that delve can support arm64 now. Now it's only a fundamental native debug, let's improve it step by step.
i didn't give up though working to fix most of the issues in one go and there are few threads at the same time regarding issues ..https://github.com/go-delve/delve/pull/1690#issuecomment-544886932

and yes let's work on it's improvement ..
Currently i am working to fix "make test" and i am very close to fix it ..

Reply:
Thank you for your understanding! There is lots work need to do next, including backtrace, fp registers, coredump. May you can help, since i have no idea for all now..

Reply:
> Thank you for your understanding! There is lots work need to do next, including backtrace, fp registers, coredump. May you can help, since i have no idea for all now..

sure ...